src/Compilers/Core/Portable/CommandLine/AnalyzerConfigSet.cs

```diff
@@ -350,4 +370,205 @@ internal static bool TryParseSeverity(string value, out ReportDiagnostic severit
             return false;
         }
     }
+
+    /// <summary>
+    /// Represents a global key that was unset due to conflicts
+    /// </summary>
+    public readonly struct AnalyzerUnsetKey
+    {
+        /// <summary>
+        /// The name of the key that was unset
+        /// </summary>
+        public string KeyName { get; }
```

Comment:
Consider making all the reference type fields as `string?`. That follows our nullable policy for `struct` members.